db/migrate_cards/20151202112547_import_migrate_tags_to_topics.rb

```diff
@@ -0,0 +1,39 @@
+# -*- encoding : utf-8 -*-
+
+def put_things_in_tag_to_correct_position cards, skip_year
```

Comment:
Assignment Branch Condition size for put_things_in_tag_to_correct_position is too high. [15.81/15]<br>Method has too many lines. [16/15]